$NetBSD: patch-af,v 1.4 2023/03/11 01:54:01 gdt Exp $

\todo Understand and probably delete.  First hunk is carried over from
before without adequate thought.  Second hunk appears to be declaring
things declared elsewhere.

--- cfs.h.orig	2013-05-15 16:50:30.000000000 +0000
+++ cfs.h
@@ -22,6 +22,8 @@
 
 #include <dirent.h>
 #include <inttypes.h>
+#include <stdlib.h> /* ? */
+#include <string.h> /* ? */
 #include <unistd.h>
 
 /* include files specific to cipher modules go here */
@@ -222,6 +224,11 @@ extern instance *instances[];
 extern nfstime roottime;
 extern cfs_fileid rootnode;
 
+void closeall(void);
+void freeinstance(int i);
+void freelist(cfs_fileid *f);
+void mask_cipher(cfskey *k, unsigned char *s, int d);
+
 #ifdef hpux
 #define seteuid(x) setresuid(-1,x,-1)
 #define setegid(x) setresgid(-1,x,-1)
